--- connection.py	2019-10-21 10:29:02.000000000 +0200
+++ connection_new.py	2019-10-21 12:00:37.425392690 +0200
@@ -76,6 +76,7 @@
         self.retry_connect = retry_connect
         self.cluster_options = cluster_options if cluster_options else {}
         ep_kwargs = dict((k, self.cluster_options[k]) for k in ('load_balancing_policy', 'retry_policy') if k in self.cluster_options)
+        del self.cluster_options['load_balancing_policy']
         try:
             ep = self.cluster_options['execution_profiles'][EXEC_PROFILE_DEFAULT]
             ep.consistency_level = self.consistency
